libs/client/ui/components/src/lib/quick-filter/quick-filter.tsx

```diff
@@ -8,17 +8,41 @@ export interface QuickFilterProps {
   className?: string;
   onChange: (values: QuickFilterOption[]) => void;
   iconSize?: number;
+  allButton?: boolean;
 }
 
 export const QuickFilter: React.FC<QuickFilterProps> = ({
   options,
   className,
   onChange,
   iconSize,
+  allButton,
 }) => {
+  const every = options?.every((o) => o.active);
+  let newOptions = options ? [...options] : null;
+  if (allButton) {
+    newOptions =
+      newOptions?.map(
+        (o) =>
+          ({
+            ...o,
+            active: every ? false : o.active,
+          } as QuickFilterOption)
+      ) ?? null;
+
+    newOptions?.unshift({
+      key: 'ALL',
```

Comment:
kama dean key vllt irgendwo in ana readonly member speichern. der wird öfters als string einfach so referenz verglicha.